packages/cmf/jest.config.js

```diff
@@ -1,8 +1,6 @@
-const path = require('path');
 const defaults = require('../../node_modules/@talend/scripts-config-jest/jest.config.js');
 
 module.exports = {
 	...defaults,
 	testRegex: '.*\\.test.js$',
-	setupFilesAfterEnv: [...defaults.setupFilesAfterEnv, path.join(__dirname, '../../test-setup.js')],
```

Comment:
This is CMF, but we remove the talend/ui test-setup that was here for i18next mock, but it's in talend/ui jest config by default.